ArgoTests/Tests/SwiftDictionaryDecodingTests.swift

```diff
@@ -6,7 +6,7 @@ class SwiftDictionaryDecodingTests: XCTestCase {
     let typesDict = [
        "numerics": [
         "int": 5,
-        "int64": 9007199254740992,
+        "int64": 900719,//9254740992, Dictionaries can't handle 64bit ints (iOS only, Mac works)
```

Comment:
I couldn't get this to compile with the full number. I was getting an error like `Integer literal '9007199254740992' overflows when stored into 'Int'` and couldn't figure out how to cast it in a way that compiled.